auditbeat/module/auditd/audit_linux.go

```diff
@@ -116,14 +115,14 @@ func (ms *MetricSet) Run(reporter mb.PushReporterV2) {
 
 	if err := ms.addRules(reporter); err != nil {
 		reporter.Error(err)
-		logp.Err("%v %v", logPrefix, err)
+		ms.log.Errorw("Failure adding audit rules", "error", err)
 		return
 	}
 
 	out, err := ms.receiveEvents(reporter.Done())
 	if err != nil {
 		reporter.Error(err)
-		logp.Err("%v %v", logPrefix, err)
+		ms.log.Errorw("Failure receiving audit events", "error", err)
```

Comment:
Would be nice if this would be `error.message` the same as we have in our `fields.yml`.

Reply:
It would. The error is written as an object, but it doesn’t match our normal structure. See https://github.com/uber-go/zap/blob/master/zapcore/error.go#L28

Reply:
It should be possible to intercept (but with some expense) the Error types and wrap them in another type that implements https://godoc.org/go.uber.org/zap/zapcore#ObjectMarshaler to customize the marshaling.

Reply:
The part I'm worried if we don't do it is that someone is going to read filebeat json logs which will conflict with our own filebeat template.